package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ainft-team/ainft-js",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
please make sure that this version is aligned as discussed.
cc: @minsulee2 @liayoo

Reply:
There was a discussion, but I was mistaken. it's fixed!